doc/requirements/management

```diff
@@ -0,0 +1,134 @@
+---
+group: Management
+---
+
+Management operations allows the user to control and maintain Open CAS.
+They are performed from console line with `casadm` management tool.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Remove active cache volume with dirty data with flush
+id: remove_cache_volume_active_flush
+---
+
+While cache volume is active and contains dirty data, the management tool shall
+allow to remove cache volume with flush.
```

Comment:
I feel like the requirements are over specified. You added requirements for removing active&dirty and inactive core but you missed one for active&clean. 

I would start with the requirement for removing core (without mentioning about any particular state) because that's the feature that CAS shall be able to handle. After this general statement I would add the requirements for flushing a dirty core before its removal, for optional skipping that step by giving additional confirmation and for interrupting the flushing.